scripts/globals/follow.lua

```diff
@@ -99,3 +99,34 @@ xi.follow.assignLeaderMod = function(mob, leaders, maxDistance)
         end
     end
 end
+
+-- This function causes the followers to organize in a column, and not
+-- stack upon one another. Must be called in leading mob's onMobRoam in
+-- order to maintain continuity of mob ordering in the line.
+--
+-- @param Original mob that leads the line of followers
+-- @param The amount of followers (assumed to be adjacent to origin mob's ID)
+-----------------------------------
+xi.follow.followLine = function(mob, numFollowers)
+    local mobID = mob:getID()
+
+    if
+        mob and
+        mob:isAlive()
+    then
+        for indexFollower = mobID + numFollowers, mobID + 1, -1 do
+            local follower = GetMobByID(indexFollower)
```

Comment:
You are using mobID + 1 but does EVERY bodyguard follow this same behavior of follower = master + 1?

If this is used outside of just this instance does all mobs that follow their master follow this rule?

Reply:
In all of my experience with following mobs, yes.
However, you're probably right in making it start based on a specific index rather than mobID + 1

Reply:
Upon review there are no non-custom mobs that would use a column follow behavior that aren't adjacent in mob ID.